app/models/application_form.rb

```diff
@@ -108,5 +108,15 @@ def within_amend_period?
     application_choices.none?(&:edit_by_expired?)
   end
 
+  def blank_application?
+    updated_at == created_at &&
+      application_choices.none &&
+      application_work_experiences.none &&
+      application_volunteering_experiences.none &&
+      application_qualifications.none &&
+      application_references.none &&
+      application_work_history_breaks.none
+  end
```

Comment:
This isn't the most elegant implementation of all time, but it was the simplest way I could think to do it.  Another option would be to add an additional column to the ApplicationForm which is updated to true if any of the above are updated.

Reply:
Hmm! What about using the [`unsubmitted_not_started_form` state](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/d9dafbafb220159ece0bf319b4f3cf7c66c95c08/app/services/process_state.rb#L19)?

Reply:
Hey, can do. I saw at. the top it says:

` the state of application choices. Do not rely on this for business logic.`

so I wasn't sure if it was okay to use it or not.

Reply:
I spoke to Tjmen and he said not to use  unsubmitted_not_started_form as various other things use it such as
 
![image](https://user-images.githubusercontent.com/42515961/76210887-f0edf180-61fc-11ea-8f90-8bda112edba6.png)

The above solves the issue anyway. I hadn't realised that objects that belong to an application form update the applications forms updated_at.